core/kits/documents/tabs/layout-settings.php

```diff
@@ -27,22 +27,11 @@ public function register_tab_controls() {
 		$this->start_controls_section(
 			'section_' . $this->get_id(),
 			[
-				'label' => $this->get_title(),
+				'label' => __( 'Layout Settings', 'elementor' ),
```

Comment:
Use `get_title` but change its return value.

Reply:
Impossible because there is a difference between the tab title and the section title

Reply:
They should be the same like all other tabs,